frontend/searching.cpp

```diff
@@ -121,7 +121,7 @@ void MainWindow::selectSimilar()
 }
 void MainWindow::on_actionSearch_triggered()
 {
-    QString textInput = QInputDialog::getText(this, tr("Enter search query"), tr("search runs for:"), QLineEdit::Normal);
+    QString textInput = QInputDialog::getText(this, tr("Find"), tr("Find in current run data (RB, user, title,...):"), QLineEdit::Normal);
```

Comment:
Question - are these the only fields that are searched in the backend? If so the ellipsis can be removed. If the list isn't much longer, those keys can be added to the list.

Reply:
It searches every currently visible column